### DISCUSSION TITLE
Ingestor and Compactor not getting Auto-forget when reached to UNHEALTHY state

### DISCUSSION THREAD

**Discussion by mdvaliyullah** [grafana/mimir#3601] [Help and support]
- Running Mimir in monolithic mode with consul config
- Observed if any pods getting evicted or removed the other components like Distributor, Ruler, Store-gateway are clearing up the unhealthy pods from the Ring list. But it's not getting cleared for Ingestor and Compactor. Please advise
- used below configuration in all the required components for Auto-forget setup so that the healthy instance removes the unhealthy instance from the ring. Please advise
```
heartbeat_period: 2s
heartbeat_timeout: 10s  
```

**Comment by mdvaliyullah** [comment]
please advise if any changes is required in the configuration

**Reply by pracucci** [reply]
Currently, ingester and compactor don't support auto-forget. We're looking for some help to add support to compactors (see [this issue](https://github.com/grafana/mimir/issues/1588)), while [here there's a discussion](https://github.com/grafana/mimir/issues/2869) whether we should support it or not for ingesters.
